### PR TITLE
Measure model integration

### PR DESCRIPTION
**Pull request by mayerm94**
Use cqm-execution with the new measure model
Pull requests into cqm-execution-service require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1607
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -67,16 +67,16 @@ app.post('/calculate', function (request, response) {
   }
 
   // Grab params from request.
-  const {measure, valueSetsByOid, patients, options = {}} = request.body
+  const {measure, valueSets, patients, options = {}} = request.body
 
-  if (Array.isArray(valueSetsByOid)){
+  if (!Array.isArray(valueSets)){
     logger.log({ level: 'error', message: 'GET /calculate. valueSets passed as array, headers: ' + JSON.stringify(request.headers) });
```
Comment (jbradl11) [inline server.js]: Update log message "ValueSets not passed as array..."
